noxfile.py

```diff
@@ -1,7 +1,22 @@
+from pathlib import Path
+
 import nox
 
 # imports all nox task provided by the toolbox
 from exasol.toolbox.nox.tasks import *
 
 # default actions to be run if nothing is explicitly specified with the -s option
 nox.options.sessions = ["project:fix"]
+
+
+@nox.session(name="update-integration-test-github-workflows", python=False)
+def update_integration_test_github_workflows(session: nox.Session):
```

Comment:
Except the actionlint, we might need the nox task also in the SLC repos

Reply:
Added a new github workflow for it, which will be deployed in the SLC repos.